src/edu/cmu/tartan/Game.java

```diff
@@ -564,9 +611,200 @@ private void help() {
 		PrintMessage.printConsole("- " + indirobj.toString() + "\n");
 		PrintMessage.printConsole("- " + misc.toString() + "\n");
 		PrintMessage.printConsole("- You can inspect an inspectable item by typing \"Inspect <item>\"\n");
+		PrintMessage.printConsole("- You can save the game by typing \"save\"\n");
+		PrintMessage.printConsole("- You can periodic save the game by typing \"save <minute>\"\n");
```

Comment:
1. Please change description: periodical -> periodically.
2. Please add the range of <minute>.